.ci/scripts/distribution/test-tck.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -xeu
+
+git clone https://github.com/zeebe-io/bpmn-tck.git
```

Comment:
Can't we release bpmn-tck versions and use mvn's packaging powers?

Reply:
See comments by Phil above